src/main/java/com/walabot/home/ble/sdk/PairingEvents.kt

```diff
@@ -0,0 +1,68 @@
+package com.walabot.home.ble.sdk
+
+import com.walabot.home.ble.Message.DevInfo
+import com.walabot.home.ble.Result
+import com.walabot.home.ble.device.DeviceInfo
+
+enum class EspPairingEvent(val value: Int) {
+    Connecting(1),
+    Connected(2),
+    GettingDevInfo(3),
+    GotDevInfo(4),
+    WifiScan(5),
+    WifiList(6),
+    WifiConnecting(7),
+    WifiConnected(8),
+    SendingCloudDetails(9),
+    SentCloudDetails(10),
+    Pairing(11),
+    Paired(12),
+    StagePairWithCloud(13),
+    NotifyPairingComplete(14),
+    CheckOTA(15),
+    PerformOTA(16),
+    CommittingProvision(17),
+    ProvisionCommited(18),
+    Rebooting(19),
+    RebootedToFactory(20),
+    ;
+
+    override fun toString(): String {
+        return when(value) {
+            1 -> "Trying to establish connection to the device"
```

Comment:
1 -> "Establishing connection to the device"
2 -> "Connected to the device"
7-> "Connecting to selected wifi"
8-> "Device has connected to wifi"
10-> "Cloud details were sent to the device"
12->"Device paired to account"